From a7533e2c9eb0554bd32723f70d64a68acb67d7b4 Mon Sep 17 00:00:00 2001
From: Jaroslav Mracek <jmracek@redhat.com>
Date: Tue, 6 Mar 2018 14:37:50 +0100
Subject: [PATCH 1/3] Reflect latest changes in libdnf

Requires: libdnf-0.13.0+

G object DnfPackageSet in libdnf was replaced by c++ class therefore
g_object_unref cannot be used anymore, but the new dnf_packageset_free has to be
used instead.
---
 backends/dnf/pk-backend-dnf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backends/dnf/pk-backend-dnf.c b/backends/dnf/pk-backend-dnf.c
index d997dee67..ddfcd4231 100644
--- a/backends/dnf/pk-backend-dnf.c
+++ b/backends/dnf/pk-backend-dnf.c
@@ -850,7 +850,7 @@ dnf_utils_run_query_with_newest_filter (DnfSack *sack, HyQuery query)
 	hy_query_free (query_tmp);
 	g_ptr_array_unref (results_tmp);
 
-	g_object_unref (pkgset);
+	dnf_packageset_free(pkgset);
 
 	return results;
 }
-- 
2.13.6

